### DISCUSSION TITLE
Need some help to plot this data

### DISCUSSION THREAD

**Discussion by ryanhanif** [pyvista/pyvista#3557] [Q&A]
I got project to create the data of the earthquake and i want to make this data had a surface (i start to looking in pyvista doc and it called interpolating) , but it need the surface data so i downloaded the surface data as tiff file (i tried to looking if possible the tiff file can also use as surface data) but i couldnt find the way how to make that tiff file use as surface data .
![Screenshot from 2022-11-05 12-56-30](https://user-images.githubusercontent.com/117502893/200104877-61c9e424-10f3-4d70-a3f3-a76aa197f988.png) (this is the data of the earthquake)
![Screenshot from 2022-11-05 12-56-25](https://user-images.githubusercontent.com/117502893/200104882-dc26715c-68b1-4ed1-bd08-1f01ecb17eba.png) (this is the tiff file)

so basically i want plot that points into the tiff file , any suggestions?

**Comment by banesullivan** [comment]
If it is a GeoTiff, you may want to use `pyvista-xarray` (with `rioxarray`) like in this example: https://github.com/pyvista/pyvista-xarray#raster-with-rioxarray

There is also this example using xarray more directly: https://banesullivan.com/pyvista/examples/raster.html

**Reply by banesullivan** [reply]
That will help you get the spatial reference/coordinates right so the points can be plotted with the surface

**Reply by ryanhanif** [reply]
so what i got from read that reference that i should "extract" the information from tiff file to get the information that i can use to make surface , is it correct sir?

**Reply by ryanhanif** [reply]
![sample](https://user-images.githubusercontent.com/117502893/200156828-7e4241f3-7461-4eb5-88a0-8f4a7274ce04.png)
so this is what it looks like what i want to plot the data .. making the points of earthquake on the surface.. 
(this sample using paraview to make the plot)
